examples/dna_language_models/dna_lm.ipynb

```diff
@@ -0,0 +1,2941 @@
+              " 'name': 'YBR063C_YBR063C_367930|0',\n",
+              " 'label': 0}"
+            ]
+          },
+          "execution_count": 15,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "raw_data['train'][0]"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "92eccf3e-e846-4c59-af56-0e336ac5a1cd",
+      "metadata": {
+        "id": "92eccf3e-e846-4c59-af56-0e336ac5a1cd"
+      },
+      "source": [
+        "We split out dataset into training, test, and validation sets."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 16,
+      "id": "f0649bbd-e74e-4dd6-a564-c4d65e46dbbf",
+      "metadata": {
+        "id": "f0649bbd-e74e-4dd6-a564-c4d65e46dbbf"
+      },
+      "outputs": [],
+      "source": [
+        "from datasets import Dataset, DatasetDict\n",
+        "\n",
+        "train_valid_split = raw_data['train'].train_test_split(test_size=0.15, seed=42)\n",
+        "\n",
+        "train_valid_split = DatasetDict({\n",
+        "    'train': train_valid_split['train'],\n",
+        "    'validation': train_valid_split['test']\n",
+        "})\n",
+        "\n",
+        "ds = DatasetDict({\n",
+        "    'train': train_valid_split['train'],\n",
+        "    'validation': train_valid_split['validation'],\n",
+        "    'test': raw_data['test']\n",
+        "})"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "5424726f-a7ba-45d5-b449-36be9a98b8e6",
+      "metadata": {
+        "id": "5424726f-a7ba-45d5-b449-36be9a98b8e6"
+      },
+      "source": [
+        "Then, we use the tokenizer and a utility function we created, ```get_kmers``` to generate the final data and labels. The ```get_kmers``` function is essential for generating overlapping 6-mers needed by the language model (LM). By using k=6 and stride=1, we ensure that the model receives continuous and overlapping subsequences, capturing the local context within the biological sequence for more effective analysis and prediction.\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 17,
+      "id": "75f267a9-82d1-4343-982e-9b1ea542a330",
+      "metadata": {
+        "id": "75f267a9-82d1-4343-982e-9b1ea542a330"
+      },
+      "outputs": [],
+      "source": [
+        "def get_kmers(seq, k=6, stride=1):\n",
+        "    return [seq[i:i + k] for i in range(0, len(seq), stride) if i + k <= len(seq)]"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 18,
+      "id": "efa9441d-f44c-4ca3-b24c-fa5c853896cd",
+      "metadata": {
+        "id": "efa9441d-f44c-4ca3-b24c-fa5c853896cd"
+      },
+      "outputs": [],
+      "source": [
+        "test_sequences = []\n",
+        "train_sequences = []\n",
+        "val_sequences = []\n",
+        "\n",
+        "dataset_limit = 200\n",
```

Comment:
Could you please add a comment that this value should be set to `None` if users want to use the whole dataset?